static/js/settings_notifications.js

```diff
@@ -81,6 +91,32 @@ exports.set_up = function () {
         });
     });
 
+    $("#play_notification_sound").click(function () {
+        $("#notifications-area").find("audio")[0].play();
+    });
+
+    $("#notification_sound").val(page_params.notification_sound);
+
+    $("#enable_sounds").change(function () {
+        if (this.checked || $("#enable_stream_sounds").is(":checked")) {
+            $("#notification_sound").prop("disabled", false);
+            $("#notification_sound").parent().removeClass("control-label-disabled");
+        } else {
+            $("#notification_sound").prop("disabled", true);
+            $("#notification_sound").parent().addClass("control-label-disabled");
+        }
+    });
+
+    $("#enable_stream_sounds").change(function () {
+        if (this.checked || $("#enable_sounds").is(":checked")) {
+            $("#notification_sound").prop("disabled", false);
+            $("#notification_sound").parent().removeClass("control-label-disabled");
+        } else {
+            $("#notification_sound").prop("disabled", true);
+            $("#notification_sound").parent().addClass("control-label-disabled");
+        }
+    });
```

Comment:
L98-118 can be simplified to this:
```js
    var notification_sound_dropdown = $("#notification_sound");
    notification_sound_dropdown.val(page_params.notification_sound);

    $("#enable_sounds, #enable_stream_sounds").change(function () {
        if ($("#enable_stream_sounds").prop("checked") || $("#enable_sounds").prop("checked")) {
            notification_sound_dropdown.prop("disabled", false);
            notification_sound_dropdown.parent().removeClass("control-label-disabled");
        } else {
            notification_sound_dropdown.prop("disabled", true);
            notification_sound_dropdown.parent().addClass("control-label-disabled");
        }
    });
```

Reply:
@shubhamdhama OK, I have simplified it. Thanks!